docs/docs.md

```diff
@@ -0,0 +1,628 @@
+# `/event/{eventKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  webcasts: {
+    type: "twitch" | "youtube"
+    url: string
+  }
+  location: {
+    name: string
+    lat: number
+    lon: number
+  }
+  name: string
+  district?: string
+  week?: number
+  // UTC date
+  startDate: string
+  // UTC date
+  endDate: string
+  // Only if user is logged in
+  starred?: false
```

Comment:
Didn't we decide we want to have a custom endpoint for starred since they're not really attached to the event but rather the user?